torchvision/models/detection/rpn.py

```diff
@@ -28,6 +28,60 @@ def _onnx_get_num_anchors_and_pre_nms_top_n(ob, orig_pre_nms_top_n):
     return num_anchors, pre_nms_top_n
 
 
+@torch.jit.unused
+def _onnx_get_grid_image_sizes(feature_maps, image_list_tensors):
+    # type: (List[Tensor], Tensor) -> Tuple[List[Tensor], Tensor]
```

Comment:
Although technically incorrect, in order for torchscript to work you'll need to make the return types in the annotation to be consistent with the ones in the other branch, following the error message in https://circleci.com/gh/pytorch/vision/73728?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

So I would instead do something like
```python
type: (List[Tensor], Tensor) -> Tuple[List[List[int]], List[int]]
```
so that `grid_sizes` and `image_sizes` have the "same type" as in the other part of the code